queries/c/fold.scm

```diff
@@ -4,4 +4,9 @@
  (while_statement)
  (translation_unit)
  (function_definition)
+ (comment)
+ (preproc_if)
+ (preproc_elif)
+ (preproc_else)
+ (preproc_ifdef)
```

Comment:
The hierarchy might be a bit of a problem if you have multiple `#else`

- if
 - else (folding the first else will fold all other else since they are children in the parse tree)
  - else
    - else